places-database-management-1/bulk-import-of-pois.md

```diff
@@ -63,15 +63,15 @@ Running the Script
 
       * `access_code`
       
-        This is your access code obtained from the call to Adobe IMS.
+        This is your access code obtained from the call to Adobe IMS. See [Prequisites for using the Places REST API](../places-rest-apis/adobe-i-o-integration/README.md) for directions on how to obtain this access code. 
```

Comment:
- In the 1st sentence, please add "that was" between "code" and "obtained".
- Please reword the 2nd sentence as follows:
"For information about how to obtain this access code, see [Prequisites for using the Places REST API](../places-rest-apis/adobe-i-o-integration/README.md) . "